src/model/AppModel.as

```diff
@@ -0,0 +1,12 @@
+package model {
+[Bindable]
+public class AppModel {
+    public var showLogs:Boolean = true;
```

Comment:
I've disabled it for now. Will restore back when task with verbose logging will be done.